testapp/yelp_clientid.yaml

```diff
@@ -0,0 +1,2 @@
+CLIENT_ID = '_tg2X_Fpcpuae2eNKaHj_w'
+CLIENT_SECRET = 'TLPjiUfjOessjQVEqQpogquNpwNHUkMrba5QCCCASgb5fUfMhwamVNvRvB9tAy78'
```

Comment:
Looks like this `.yaml` file still got included in the git history. Can you remove it and fixup all the commits into one commit?